src/test/scala/logger/LoggerSpec.scala

```diff
@@ -8,14 +12,29 @@ import Converter.map2JValue
 
 case class Poney(name: String, age: Int)
 
-class LoggerSpec extends FlatSpec  {
+class LoggerSpec extends FlatSpec {
+
+  def hideSTDOut(testCode: (ByteArrayOutputStream) => Unit) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use : Unit = for procedures [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ProcedureDeclarationChecker)